cmd/tendermint/commands/init.go

```diff
@@ -99,5 +106,87 @@ func initFilesWithConfig(config *cfg.Config) error {
 		logger.Info("Generated genesis file", "path", genFile)
 	}
 
+	if err := InitIpfs(config); err != nil {
+		return err
+	}
+
 	return nil
 }
+
+// TODO move into separate file
+func InitIpfs(config *cfg.Config) error { // add counter part in ResetAllCmd
+	// init IPFS config with params from config.IPFS
+	// and store in config.IPFS.ConfigRootPath
```

Comment:
Nit: what does `// add counter part in ResetAllCmd` mean? Is it a TODO? The location is just weird, to the right of the function decl.

For the rest of the comment lines here, can these be moved above the function decl?

Reply:
No, it's simply a description of what happens internally: it takes the (few) ipfs related config flags from the tendermint config and applies them to the ipfs config.

Reply:
Yeah, this should indeed be a TODO: tendermint has a sub-command `unsafe-reset-all`, which:
>  (unsafe) Remove all the data and WAL, reset this node's validator to genesis state

I guess, we should probably delete the ipfs data (block store), too? That would be identical of deleting all chunks that node sampled so far. I'm not sure yet. Let's make it a todo.